### PR TITLE
Translated README.md into Japanese

### PR DESCRIPTION
**Pull request by uta-mori**
Thank you for the great library.

I've translated it into Japanese for my own study and for the sake of Japanese developers to know about this library.

### CODE REVIEW

File: README_jp.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+fun main() {
+    val database = Database.connect("jdbc:mysql://localhost:3306/ktorm?user=root&password=***")
+
+    for (row in database.from(Employees).select()) {
+        println(row[Employees.name])
+    }
+}
+````
+
+Now you can run this program, Ktorm will generate a SQL `select * from t_employee`, selecting all employees in the table and printing their names. You can use the for-each loop because the query object returned by the `select` function implements the `Iterable<T>` interface. Any other extension functions on `Iterable<T>` are also available, eg. map/filter/reduce provided by Kotlin standard lib.
```
Comment (vincentlauvlwj) [inline README_jp.md:93]: Why there is still an English sentence here?

File: README_jp.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+
+
+これで列バインディングが設定されたので、[sequence APIs](#Entity-Sequence-APIs)を使ってエンティティに対して多くの操作を行うことができます。以下のコードのように、まず `sequenceOf` でシーケンスオブジェクトを作成し、次に `find` 関数を呼び出して`vince`と言う名前の従業員情報を取得します。
+
+```kotlin
+val sequence = database.sequenceOf(Employees)
+val employee = sequence.find { it.name eq "vince" }
+```
+
+We can also filter the sequence by the function `filter`. For example, obtaining all the employees whose names are vince:
```
Comment (vincentlauvlwj) [inline README_jp.md:277]: Here, still English.

File: README_jp.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+
+## エンティティシーケンス API
+
+Ktormは*Entity Sequence*という名前のAPIセットを提供しており、これを使ってデータベースからエンティティオブジェクトを取得することができます。その名が示すように、そのスタイルや利用パターンはKotlin標準ライブラリのシーケンスAPIと非常によく似ており、`filter`, `map`, `reduce`などの同じ名前の拡張関数が多数提供されています。
+
+エンティティシーケンスAPIのほとんどは拡張関数として提供されており、それらは`中間処理`と`末端処理`の2つのグループに分けることができます。
+
+### 中間処理
+
+These functions don’t execute the internal queries but return new-created sequence objects applying some modifications. For example, the `filter` function creates a new sequence object with the filter condition given by its parameter. The following code obtains all the employees in department 1 by using `filter`:
```
Comment (vincentlauvlwj) [inline README_jp.md:334]: Still English.

Comment (uta-mori) [reply README_jp.md]: I'm sorry. I forgot to erase it.

**Pull request merged**